extensions/mssql/src/tableDesigner/tableDesigner.ts

```diff
@@ -27,11 +27,11 @@ export function registerTableDesignerCommands(appContext: AppContext) {
 				throw new Error(FailedToGetConnectionStringError);
 			}
 			let titleString = `${context.connectionProfile!.serverName} - ${context.connectionProfile!.databaseName} - ${NewTableText}`;
-			// append non default options to end to let users know exact connection.
-			let nonDefaultOptions = await azdata.connection.getEditorConnectionProfileTitle(context.connectionProfile, true);
-			nonDefaultOptions = nonDefaultOptions.replace('(', '[').replace(')', ']');
-			if (nonDefaultOptions !== '') {
-				titleString += `${nonDefaultOptions}`;
+			// append distinguishing options to end to let users know exact connection.
+			let distinguishingOptions = await azdata.connection.getEditorConnectionProfileTitle(context.connectionProfile, true, false);
+			if (distinguishingOptions !== '') {
```

Comment:
Well in the case of table Designer, STS expects the title to be in a certain format (That does not take into account Connection Name, which isn't an option we count as advanced options) and it will overwrite the result once Table Designer is connected. Since we cannot just use the title as is in this scenario 

Other areas such as Connection Status and the Query Editor currently use the title in full, there will need to be some rewrites on the STS side to account for this, but I will have to discuss with Alan about this first.